CheatSheets/git-cheatsheet.md

```diff
@@ -92,6 +92,8 @@ Table of Contents
 | `git push origin [branch name]` | Push a branch to your remote repository |
 | `git pull origin [branch name]` | Pull a branch from your remote repository |
 | `git remote origin [branch name]` | Connect repository to local server |
+| `git push -f` | force a push request
```

Comment:
```suggestion
| `git push <remote> --force` | Forces the push even if it results in a non-fast-forward merge. Be sure that nobody has pulled the commits before using the `--force` option. |
```
| `git push <remote> --force` | Forces the push even if it results in a non-fast-forward merge. Be sure that nobody has pulled the commits before using the `--force` option. |

Need to `mention the <remote> branch`.

Reply:
Ok. @crescentpartha 

Thanks for Suggestion. I am changing it.